gradle.properties

```diff
@@ -16,6 +18,8 @@ org.gradle.java.installations.auto-detect=false
 
 # log some dependency verification info to console
 org.gradle.dependency.verification.console=verbose
+# fixme undo lenient
+org.gradle.dependency.verification=lenient
```

Comment:
why does 

`gradle --write-verification-metadata sha256`

not include dependencies ?